python/HeaderService/hutils.py

```diff
@@ -310,22 +326,22 @@ def load_templates(self):
         self.header_primary = read_head_template(self.templ_primary_file)
 
         # Load up the template for the PRIMARY header
-        LOGGER.info("Loading template for: {}".format('PRIMARY'))
+        self.log.info("Loading template for: {}".format('PRIMARY'))
         self.header['PRIMARY'] = self.header_primary
 
         # Update PRIMARY with new value in self.CCDGEOM
-        LOGGER.debug("Updating GEOM in template for: {}".format('PRIMARY'))
+        self.log.debug("Updating GEOM in template for: {}".format('PRIMARY'))
         PRIMARY_DATA = self.CCDGEOM.get_extension('PRIMARY')
         self.update_records(PRIMARY_DATA, 'PRIMARY')
 
         # For the Segments, we load it once and then copy and
         # modify each segment
         for SEG in self.segment_names:
             EXTNAME = '{}{}'.format(self.segname, format(SEG))
-            LOGGER.info("Loading template for: {}".format(EXTNAME))
+            self.log.info("Loading template for: {}".format(EXTNAME))
             self.header[EXTNAME] = copy.deepcopy(self.header_segment)
             # Now get the new value for the SEGMENT
-            LOGGER.debug("Updating GEOM in template for: {}".format(SEG))
+            self.log.debug("Updating GEOM in template for: {}".format(SEG))
```

Comment:
Consider using f strings instead of format, for readability. For example (also avoiding all caps):
```
for seg in self.segment_names:
    extname = self.segname + format(seg)  # or f"{self.segname}{seg}"
    self.log.info(f"Loading template for {extname}")
    self.header[extname] = copy.deepcopy(self.header_segment)
    # Now get the new value for the segment
    self.log.debug(f"Updating GEOM in template for {seg}")
```

Reply:
Will do. Thanks.